docs/collections/documents/tutorials/using_rest_api.md

```diff
@@ -1608,10 +1634,10 @@ print("\nStream deleted: ", result)
   const FEDERATION_NAME = "api-gdn.prod.macrometa.io";
   const FEDERATION_URL = `https://${FEDERATION_NAME}`;
 
-  const IS_GLOBAL = true;
+  const IS_GLOBAL = false;
   const STREAM_NAME = `tutorialAppInputStream`;
-  const STREAM_APP_NAME = `strean_app_tutorial`;
-  const STREAM_APP = `@App:name('strean_app_tutorial')
+  const STREAM_APP_NAME = `stream_app_tutorial`;
+  const STREAM_APP = `@App:name('stream_app_tutorial')
     @App:description('This application demonstrates how to use user defined function in the stream app')
```

Comment:
```suggestion
    @App:description('This application demonstrates how to use user-defined functions in a stream app')
```

Reply:
Here, stream app is two words. Elsewhere, it is one. Please choose one and stick with it.